Sources/OpenAPIRuntime/Conversion/Converter.swift

```diff
@@ -12,11 +12,11 @@
 //
 //===----------------------------------------------------------------------===//
 #if canImport(Darwin)
-import Foundation
+import class Foundation.JSONEncoder
 #else
 @preconcurrency import class Foundation.JSONEncoder
-@preconcurrency import class Foundation.JSONDecoder
 #endif
```

Comment:
The funny thing is `JSONEncoder` is an `open class` so it can't be `Sendable` at all and the Linux implementation is the correct one here.